students/colephalen/lesson01/assignment/tests/test_unit.py

```diff
@@ -0,0 +1,32 @@
+"""
+Unit Tests for all modules and classes from the inventory_management dir
```

Comment:
This is a great start to your tests! Try to add this for the other classes such as inventory and Furniture.